frontend/src/components/eMIB/OrganizationalStructure.jsx

```diff
@@ -47,95 +50,190 @@ class OrganizationalStructure extends Component {
+      this.populateTreeArray(response);
+      // saving the organizational structure markdown and tree view content in local states
       this.setState({
         markdown_en: response.background.en.background[0].markdown[2].text,
-        markdown_fr: response.background.fr.background[0].markdown[2].text
+        markdown_fr: response.background.fr.background[0].markdown[2].text,
+        isLoadingComplete: true
       });
     });
   };
 
   render() {
     const { currentLanguage } = this.props;
-    const treeView = [
-      {
-        id: 0,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.president,
-        groups: [1, 5, 6, 7],
-        level: 1
-      },
-      {
-        id: 1,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.corpDirector,
-        parent: 0,
-        groups: [2, 3, 4],
-        level: 2
-      },
-      {
-        id: 2,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.hr,
-        parent: 1,
-        level: 3
-      },
-      {
-        id: 3,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.finance,
-        parent: 1,
-        level: 3
-      },
-      {
-        id: 4,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.it,
-        parent: 1,
-        level: 3
-      },
-      {
-        id: 5,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.research,
-        parent: 0,
-        level: 2
-      },
-      {
-        id: 6,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.programDev,
-        parent: 0,
-        level: 2
-      },
-      {
-        id: 7,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.communications,
-        parent: 0,
-        groups: [8, 9, 10, 11],
-        level: 2
-      },
-      {
-        id: 8,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.qa,
-        parent: 7,
-        level: 3
-      },
-      {
-        id: 9,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.services,
-        parent: 7,
-        level: 3
-      },
-      {
-        id: 10,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.audits,
-        parent: 7,
-        level: 3
-      },
-      {
-        id: 11,
-        name: LOCALIZE.emibTest.background.organizationalStructure.dialog.training,
-        parent: 7,
-        level: 3
-      }
-    ];
+    const { treeViewContent_en, treeViewContent_fr } = this.state;
+    let treeView = [];
+    // waiting for tree view content data loading
+    if (this.state.isLoadingComplete) {
+      treeView = [
+        {
+          id: 0,
+          name: `${
+            currentLanguage === LANGUAGES.english
+              ? treeViewContent_en.text
+              : treeViewContent_fr.text
+          }`,
+          groups: [1, 5, 6, 7],
+          level: 1
```

Comment:
Comment2 I feel like this list should be created when you are looping over the treeviews in populateTreeArray (see Comment1)
That will be more future proof, since this will break when you try to implement pizza
groups and level should be calculated not hardcoded.
I'm ok with this being changed in a separate PR (since this a large one as it is), but it will need to be done at some point

Reply:
talked offline, we will go ahead with the hybrid load from DB/hard coding for now and shift to building treeView entirely from the DB in a future PR

Reply:
I agree with this - this isn't dynamic loading, so we'll still need to do this in the future because each tree view with have a different structure.